e2e/src/features/multimedia.feature

```diff
@@ -0,0 +1,17 @@
+@MediaViewer
+Feature: Multimedia Video E2E
+
+  Background:
+    Given I am on Media Viewer Page
+    Then I expect the page header to be "MEDIA VIEWER DEMO"
+    When the user selects the multimedia option
+
+  @EM-4000 @play
+  Scenario: Check whether the video player is working or not?
+    And the user selects play option
+    Then I should see video should be in play mode
```

Comment:
this should probably be "I should see video in play mode"

Reply:
fixed